build/Elasticsearch.Net.Virtual.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+	<metadata>
+		<id>Elasticsearch.Net.Virtual</id>
```

Comment:
Naming is hard 😃 

`Elasticsearch.Net.Virtual` feels somewhat vague. The package is a way to mock/stub a cluster, with a specific `IConnection` implementation. Would `Elasticsearch.Net.VirtualCluster`, or  `Elasticsearch.Net.MockCluster` or `Elasticsearch.Net.StubCluster` or `Elasticsearch.Net.FakeCluster` better convey the usage?